Octokit/Clients/AuthorizationsClient.cs

```diff
@@ -36,7 +36,7 @@ public AuthorizationsClient(IApiConnection apiConnection) : base(apiConnection)
         /// <returns>A list of <see cref="Authorization"/>s.</returns>
         public Task<IReadOnlyList<Authorization>> GetAll()
         {
-            return ApiConnection.GetAll<Authorization>(ApiUrls.Authorizations(), null);
+            return ApiConnection.GetAll<Authorization>(ApiUrls.Authorizations(), (IDictionary<string,string>)null);
```

Comment:
I shouldn't *need* to pass `null` in here. I'll investigate further and see what other options I have here.